lib/plug/test.ex

```diff
@@ -96,8 +96,17 @@ defmodule Plug.Test do
   @doc """
   Return the assets that have been pushed.
   """
-  def sent_pushes(%Conn{adapter: {Plug.Adapters.Test.Conn, %{pushes: pushes}}}) do
-    Enum.reverse(pushes)
+  def sent_pushes(%Conn{adapter: {Plug.Adapters.Test.Conn, %{ref: ref}}}) do
```

Comment:
We need to document this function works as a side-effect. So if you call it twice, you will get an empty list.